moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -186,34 +189,68 @@ const DashboardView = () => {
       title: "Project name",
       field: "project.project_name",
       editable: "never",
-      cellStyle: { ...typographyStyle, minWidth: "200px" },
+      cellStyle: { ...typographyStyle },
       render: entry => (
         <RenderFieldLink
           projectId={entry.project_id}
           value={entry.project_name}
         />
       ),
+      width: "25%",
```

Comment:
Went ahead and updated the column widths to reflect the changes @chiaberry made in the milestones table